ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
> Presumably the part of this PR that loads the new root CA into the security config will be need to be changed at that point, so that managers renewing their keys don't overwrite the special transitional rootCA object that I'm positing above.

I need to verify this, but I think that the manager that is the leader and hence CA will not overwrite the rootCA that it is using to sign certs because if a `rootCA` is a signer, it will issue itself new certs rather than call a remote API, which won't trigger this particular code path.

If that's the case, then I think I also need to make sure that when the CA server stops (if a node loses leadership for instance) it switches back to a signer-less `RootCA` object, otherwise it will never get root certificate updates when requesting new TLS certs.